frontend/src/components/TE/LastVersionMixin.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Vue } from "vue-property-decorator";
+import _ from "lodash";
+
+@Component
+export default class LastVersionMixin extends Vue {
+  get version() {
+    return (
+      this.item.versions &&
+      _.get(this.item, `versions[${this.item.versions.length - 1}]`, false)
```

Comment:
I don't see the justification of using `lodash` here. Can you explain why we can't just use `this.item.versions[this.item.versions.length - 1]`?